docs/introduction/console-commands-for-application-management-phing-targets.md

```diff
@@ -166,9 +166,34 @@ Runs background jobs. Should be executed periodically by system Cron every 5 min
 
 Essential for the production environment. Periodically executed Cron modules recalculate visibility, generate XML feeds and sitemaps, provide error reporting etc.
 
+If you want to have more cron instances registered, you need to create new targets with instance specified.  
+For example:
```

Comment:
I would maybe omit the example and just link the new cookbook here. (Something like "For more information, see Working with multiple..."

Reply:
I would probably link both articles - general information about cron, as well as the cookbook :)

Reply:
I've updated links below :)